README.md

```diff
@@ -16,17 +14,22 @@ machine:
 
 dependencies:
   post:
-    - curl -L https://s3.amazonaws.com/codeclimate/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+    - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
     - chmod +x ./cc-test-reporter
 
 test:
   pre:
     - ./cc-test-reporter before-build
 
   post:
-    - ./cc-test-reporter after-build --exit-code $CIRCLE_EXIT_CODE
+    - ./cc-test-reporter after-build --exit-code $TEST_SUITE_EXIT_CODE
 ```
 
+  Where:
+
+  - `$TEST_EXIT_CODE` should be the exit code when the test suite ran. Some CI
+    services don't expose directly a var with the
```

Comment:
I agree with `EXIT_CODE`? instead.

The documentation is tricky here, and I think we want to provide examples of the two ways I can think of to do it. One is with a provided env var, and the other is capturing the exit code as you mentioned.

* Travis CI: `./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT ` (Note that `$? [should _not_](https://github.com/travis-ci/travis-ci/issues/3771) be used in Travis CI)
* Circle, Jenkins: manual way. Can we cite a (tested) example (our app I guess)? Also, I saw linux `pipefail` could be useful in some cases if you're 1-lining it but guess we don't want to encourage that.